openshift/helpers.sh

```diff
@@ -132,8 +123,9 @@ function wait_for_rds_instance_info() {
     while true; do
         echo "Trying to get RDS DB endpoint for ${RDS_INSTANCE_NAME} ..."
 
-        RDS_ENDPOINT=$(get_rds_instance_info | grep -w Address | awk '{print $4}')
-        RDS_ARN=$(get_rds_instance_info | grep -w DBInstanceArn | awk '{print $4}')
+        get_rds_instance_info
```

Comment:
It is better to use native Linux utilities in shell scripts than relying on json parsing. It is possible that jq may not be installed for everyone.

Reply:
Ain't we already rely on non standard binaries like aws-cli, passwd-gen,..etc.?

Reply:
We can achieve the same without too much complexity using native utilities. We should utilise them as much as possible. This is a POV, I leave it to you.